packages/domi/test/MDArray/MDArrayRCP_UnitTests.cpp

```diff
@@ -578,8 +580,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( MDArrayRCP, toString3D, T )
 
 UNIT_TEST_GROUP(int)
 #if 1
-UNIT_TEST_GROUP(long)
-UNIT_TEST_GROUP(float)
+UNIT_TEST_GROUP(long_long_type)
```

Comment:
Do these integers correspond to Tpetra's GlobalOrdinal template parameter?  If so, then you'll want to use Tpetra's macros to decide whether those types are enabled.  Even if ETI is OFF, users are not allowed to use disabled types.  See `TpetraCore_config.h` (generated in the build directory, in `packages/tpetra/core/src/` for the macros; they have the form `HAVE_TPETRA_INST_INT_LONG` (replace `LONG` with e.g., `LONG_LONG` or `INT`, for the GlobalOrdinal type).

Reply:
No, these are `SCALAR` types.  The ordinal types are determined at compile time and default to `int` for multidimensional indexes and `long long` for 1D flattened dimensions.

Reply:
Note that even if ETI is OFF, you must first check the macros before using a particular integer type as a GlobalOrdinal type in Tpetra.